README.md

```diff
@@ -1,2 +1,95 @@
-# whokipedia
-Test your knowledge of the who's who in Whokipedia, where each clue leads you closer to uncovering the celebrity behind the blur.
+<p align="center">
+  <img src="https://github.com/roxannecvl/whokipedia/assets/125833841/be103f3e-8209-44b2-9df7-47c5b227d789" alt="image" width="300">
+</p>
```

Comment:
I suggest adding a background for the logo or taking a version that does not have a transparent background because it is hardly visible on a dark background